### ISSUE TITLE
JS error in Admin-Form  on checkboxes multiWidgets.jsx

### ISSUE THREAD

**Issue by j-b-grenouille** [lektor/lektor#894]
Lektor, version 3.2.0

Setup:

add checkboxes data field to model

```
[fields.test]
label = test
type = checkboxes
choices = trick, track, truck
```

the chekboxes are dispalyed correct, but if you try to check them they do not work and you'll  get following error at the browser console

same if you use **_source_** instad of **_choices_**

error differs from #889

```
Uncaught TypeError: this is undefined
    onChangeHandler multiWidgets.jsx:47
    onChange multiWidgets.jsx:58
    React 17
        callCallback
        invokeGuardedCallbackDev
        invokeGuardedCallback
        invokeGuardedCallbackAndCatchFirstError
        executeDispatch
        executeDispatchesInOrder
        executeDispatchesAndRelease
        executeDispatchesAndReleaseTopLevel
        forEachAccumulated
        runEventsInBatch
        runExtractedPluginEventsInBatch
        handleTopLevel
        batchedEventUpdates$1
        batchedEventUpdates
        dispatchEventForLegacyPluginEventSystem
        attemptToDispatchEvent
        dispatchEvent
    unstable_runWithPriority scheduler.development.js:653
    React 4
        runWithPriority$1
        discreteUpdates$1
        discreteUpdates
        dispatchDiscreteEvent

```

if you add the values by hand to contents.lr

```
--- 
test: trick, truck


```
the checkboxes are checked but still not editable

- Version 3.1.3 works, but throws a lot of errors discribing warnings
- current Master (5a2895c1b683c1df9) does not build **_make build-js_**, so I can't test

**Comment by abruening-iwt** [comment]
This happened to me while following the categories guide from the website (https://www.getlektor.com/docs/guides/categories/). It works fine in current Master.

**Comment by yagebu** [comment]
This should be fixed 3.2.1.
